src/contexts/note-context.js

```diff
@@ -14,16 +14,33 @@ export const NoteProvider = ({children}) => {
     }
     const [ note , setNote ] = useState(defaultState);
     const [ notes , setNotes ] = useState([]);
+    const [deletedNotes, setDeletedNotes ] = useState([]);
 
     const saveNote = async () => {
         // eslint-disable-next-line eqeqeq
-        if(note == defaultState) return;
+        console.log(note);
+        if(JSON.stringify(note)  === JSON.stringify(defaultState)) return;
+        console.log(`Saving...${note}`);
```

Comment:
remove console if not needed

Reply:
Will remove the console logs at the end of development. Need it for now.